app/models/concerns/foreman_openscap/policy_extensions.rb

```diff
@@ -244,5 +251,28 @@ def populate_overrides(puppetclass, hostgroup)
         end
       end
     end
+
+    def set_certificates_overrides(puppetclass)
+      [CA_FILE_PARAMETER, HOST_CERTIFICATE_PARAMETER, HOST_PRIVATE_KEY_PARAMETER].each do |override_parameter|
+        unless certificate_param = puppetclass.class_params.where(:key => override_parameter).first
+          errors[:base] << _("Puppet class %{class} does not have %{parameter} class parameter.") % {:class => SCAP_PUPPET_CLASS, :parameter => override_parameter}
+          return false
+        end
+        certificate_param.override = true
+        case override_parameter
+          when CA_FILE_PARAMETER
+            certificate_param.default_value = '/etc/rhsm/ca/katello-server-ca.pem'
```

Comment:
Why not? They're authenticating with the RHSM certificates; they should use the canonical source for that information, which is `/etc/rhsm/rhsm.conf`.  katello-agent does the same, although that is more closely linked to rhsm.

For whatever reason, the certificate was renamed from `/etc/rhsm/ca/candlepin-local.pem` to  `/etc/rhsm/ca/katello-server-ca.pem`, so older clients may have the older cert location.

If you don't read from rhsm.conf, you'd be forcing users to regenerate all of their certificate RPMs and redeploy them with the newer names, or change all of their older clients some other way? I don't think that's a reasonable ask.

The client knows where its certificates are, why is it being sent from the Foreman server anyway?